apps/remix-ide/src/app/plugins/solidity-umlgen.tsx

```diff
@@ -136,12 +136,14 @@ export class SolidityUmlGen extends ViewPlugin implements ISolidityUmlGen {
    * @returns {Promise<string>}
    */
   async flattenContract (source: any, filePath: string, data: any) {
+    let filename = filePath.split('/')[1].split('.')[0]
     const dependencyGraph = getDependencyGraph(data.sources, filePath)
     const sorted = dependencyGraph.isEmpty()
         ? [filePath]
         : dependencyGraph.sort().reverse()
     const result = concatSourceFiles(sorted, source.sources)
-    await this.call('fileManager', 'writeFile', `${filePath}_flattened.sol`, result)
+    await this.call('fileManager', 'writeFile', `contracts/${filename}_flattened.sol`, result)
```

Comment:
Why you are putting the path `contracts/` with the file name?

Reply:
if I don't it saves the flattened contract in the root of file explorer